libs/build_metplus.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash                                                                                                                              
+
+set -eux
+
+name="METplus"
+version=${1:-${STACK_metplus_version}}
+
+cd  ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+software="v"$version
+pkg_name=$name-$version
+met_version=${1:-${STACK_metplus_met_version}}
+url="https://github.com/dtcenter/METplus/archive/$software.tar.gz"
+[[ -d $software ]] || ( $WGET $url; tar -xf $software.tar.gz )
+[[ -d $pkg_name ]] && cd $pkg_name || ( echo "$pkg_name does not exist, ABORT!"; exit 1 )
+
+
+# Hyphenated version used for install prefix
+compiler=$(echo $HPC_COMPILER | sed 's/\//-/g')
+mpi=$(echo $HPC_MPI | sed 's/\//-/g')
+
+if $MODULES; then
+    prefix="${PREFIX:-"/opt/modules"}/$compiler/$name/$version"
+    mkdir -p $prefix
+    met_prefix="${PREFIX:-"/opt/modules"}/$compiler/$mpi"
```

Comment:
This is going to evaluate differently than the previous line.
I think you want `met_prefix=$prefix`